addon/components/line-chart/component.js

```diff
@@ -158,7 +158,9 @@ export default BaseChartComponent.extend({
                         const value = this.get('data')[d.data.key][i];
                         const formattedValue = formatter(value);
                         const secondaryClass = d.y === value ? 'primary-stat' : '';
-                        str = str.concat(`<span class="tooltip-list-item"><span class="tooltip-label ${secondaryClass}">${title}</span><span class="tooltip-value ${secondaryClass}">${formattedValue}</span></span>`);
+                        const replaceTag = title.replaceAll('<', '&lt;');
+                        const formattedTitle = replaceTag.replaceAll('>', '&gt;');
```

Comment:
Looking at the docs on replaceAll, it looks like it's only just gaining support in Chrome (Chrome 85 is the most current). Maybe we could use `replace` instead
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll

```suggestion
                        const formattedTitle = replaceTag.replace(/>/g, '&gt;');
                        const formattedTitle = replaceTag.replaceAll('>', '&gt;');
```